lib/ansible/modules/cloud/atomic/atomic_image.py

```diff
@@ -53,6 +53,12 @@
         required: False
         choices: ["yes", "no"]
         default: yes
+    storage:
```

Comment:
This is where the image is pulled to, it is the destination of the image.  I've renamed it (in both modules) to "backend" that specifies the backend to use for managing it.